dts/arm/yaml/st,stm32-rcc.yaml

```diff
@@ -0,0 +1,28 @@
+---
+title: STM32 RCC
+id: st,stm32-rcc
+version: 0.1
+
+description: >
+    This binding gives a base representation of the STM32 Clock control
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "st,stm32-rcc"
+
+  - reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+cell_string: clocks
+
+"#cells":
+  - bus
```

Comment:
I can have a try but I don't know if script will handle it.
It's a bit picky with clocks node (I'll fw you a mail about that)